src/textOnlyPackages/src/microsoft.netcore.platforms/3.1.4/microsoft.netcore.platforms.nuspec

```diff
@@ -0,0 +1,21 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/01/nuspec.xsd">
+  <metadata minClientVersion="2.12">
+    <id>Microsoft.NETCore.Platforms</id>
+    <version>3.1.4</version>
+    <title>Microsoft.NETCore.Platforms</title>
+    <authors>Microsoft</authors>
+    <owners>microsoft,dotnetframework</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="expression">MIT</license>
+    <licenseUrl>https://licenses.nuget.org/MIT</licenseUrl>
+    <icon>Icon.png</icon>
```

Comment:
This should be removed along with the icon file

Reply:
Yeah - I removed both and the icon file and it's still failing, even after cleaning with `git clean -xdff`. I'll add the DependencyPackageProjects entry and try again.

Reply:
are you building via `./build.sh --sourcebuild`?

Reply:
I've copied the build command from the CI build log: `./build.sh --ci --configuration Release --restore --build --pack --publish -bl /p:SourceBuildNonPortable= /p:ArcadeBuildFromSource=true /p:AssetManifestFileName=SourceBuild_Managed.xml`

Reply:
Besides the following:
```
    <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\System.Drawing.Common.4.7.2.csproj" />
```
should I also add these two to `eng/Build.props`?
```
    <DependencyPackageProjects Include="$(RepoRoot)src\textOnlyPackages\src\**\Microsoft.NETCore.Platforms.3.1.4.csproj" />
    <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\System.Windows.Extensions.4.7.0.csproj" />
```
I'm adding `Microsoft.NETCore.Platforms.3.1.4` as `System.Drawing.Common.4.7.2` depends on it - maybe I could have modified that dependency to use the old package `Microsoft.NETCore.Platforms.3.1.0`.

Since I'm modifying `System.Windows.Extensions.4.7.0` to update `System.Drawing.Common` dependency to `4.7.2`, I thought I might need to add it to `DependencyPackageProjects`.

Reply:
Fixed the issues and pushed new changes.